applications/notifications/controllers.py

```diff
@@ -0,0 +1,113 @@
+from datetime import datetime
+
+from applications.notifications.models import *
+import applications.notifications.dal as db
+from graphspace.exceptions import ErrorCodes, BadRequest
+from graphspace.database import *
+
+from django.conf import settings
+
+
+# Function to get notification count
+def get_notification_count(request, owner_email=None, is_read=None):
+    count = db.get_notification_count(
+        request.db_session, owner_email=owner_email, is_read=is_read)
+    return count
+
+# Function to add owner notification to database
+
+
+def add_owner_notification(message, type, resource, resource_id, owner_email=None, is_read=False, is_email_sent=False):
+    sess = Database().session()
+    notify = db.add_owner_notification(sess, message=message, type=type, owner_email=owner_email,
+                                       resource=resource, resource_id=resource_id, is_read=is_read, is_email_sent=is_email_sent)
+    # Apply changes to DB as the Middleware is not called
+    sess.commit()
```

Comment:
@melvin15may what do you mean by `# Apply changes to DB as the Middleware is not called` ?